src/vanilla/utils/devtools.ts

```diff
@@ -1,5 +1,5 @@
+import type { EnhancerOptions } from '@redux-devtools/extension'
```

Comment:
This produces types in dist. We would need to add it a peer dep.
There is a previous discussion, but the solution is probably copying&pasting types. Or just use `unknown`.

Reply:
Indeed. I would lean towards peer dep, it can't hurt...